spec/models/recipe_spec.rb

```diff
@@ -12,4 +12,16 @@
     it {should have_many :recipe_ingredients}
     it {should have_many(:ingredients).through(:recipe_ingredients)}
   end
+
+  describe ".instance methods" do
+    describe ".total_cost" do
+      it "calculates the total cost of ingredients for the recipe" do
+        ingredient_1 = Ingredient.create!(name: "BBQ", cost: 2)
+        ingredient_2 = Ingredient.create!(name: "Chicken", cost: 12)
+        recipe_1 = Recipe.create!(name: "BBQ Chicken", complexity: 2, genre: "Soul food", ingredients: [ingredient_1, ingredient_2])
+
+        expect(recipe_1.total_cost).to eq(14)
+      end
+    end
+  end
```

Comment:
```ruby
def total_cost
  14
end
```

Does your assertion disprove that this is what your method looks like? Probably not. My suggestion, for instance methods, is to test more than 1 instance object so that way your code proves that the value isn't hard-coded and is dynamic.

```ruby
expect(recipe_1.total_cost).to eq(14)
expect(recipe_2.total_cost).to eq(12) # create another recipe
```